src/test/ui/consts/duration-consts-2.rs

```diff
@@ -50,6 +51,15 @@ fn duration() {
 
     const ONE_AS_F64 : f64 = ONE.div_duration_f64(ONE);
     assert_eq!(ONE_AS_F64, 1.0_f64);
+
+    const MAX_SATURATING_ADD_ONE : Duration = MAX.saturating_add(ONE);
+    assert_eq!(MAX_SATURATING_ADD_ONE, MAX);
+
+    const ZERO_SATURATING_SUB_ONE : Duration = ZERO.saturating_sub(ONE);
+    assert_eq!(ZERO_SATURATING_SUB_ONE, ZERO);
+
+    const MAX_SATURATING_MUL_TWO : Duration = MAX.saturating_mul(2);
```

Comment:
You should maintain consistency with the previous lines, but the space before `:` isn't the standard format. I wonder why this file hasn't been through rustfmt yet.

Reply:
Its going to be moved into libstd unit tests soon, I think the formatting will happen at that point.

Reply:
See https://github.com/rust-lang/rust/pull/76335